### PR TITLE
update must gather command, based on recent changes

### PR DESCRIPTION
**Pull request by dbasunag**

### CODE REVIEW

**Comment by dbasunag** [comment]
!verified

**Comment by dbasunag** [comment]
!verified

**Comment by dbasunag** [comment]
!verified

**Review by rnetser — APPROVED** [review]

**Comment by rnetser** [comment]
!verified

File: ocp_utilities/must_gather.py [hunk 1]
```diff
@@ -37,8 +39,10 @@ def run_must_gather(
         base_command += " --insecure-skip-tls-verify"
     if kubeconfig:
         base_command += f" --kubeconfig {kubeconfig}"
-    # script_name must be the last argument
     if script_name:
         base_command += f" -- {script_name}"
-
+    # flag_name must be the last argument
```
[the remark this answers is not in the document]
Comment (dbasunag) [reply ocp_utilities/must_gather.py:48]: @rnetser one of them can't be the last. If flag_name is passed, it must be the last one. The link to must-gather changes were added to `More details` section.

**Comment by rnetser** [comment]
!cherry-pick v4.12

**Pull request merged**